src/index.js

```diff
@@ -148,6 +150,7 @@ export default class TrackingLink extends Component {
 
     return (
       <a
+        aria-label={ariaLabel || ''}
```

Comment:
this will seem nitpicky but when doing optional inputs to a function, it's more ideal to set a sensible default in the parameter list (or at the top of the function) rather than inline when you're using the value.

when doing layout stuff (outputting HTML), it's also preferable to put as little decision-making logic into the rendering part and moving that logic up out of the render so that you can be as declarative as possible in the view layer.

there's a few reasons for this, but it all boils down to rigorous application of the Single Responsibility Principle. The responsibility of the `return` function of a component is to render the component, not make decisions about whether a given variable is valid. If, for instance, `ariaLabel` were used in more than one place in this component, you'd need to repeat this check to see if needs a default value. We're not doing it multiple times here, of course, but the pattern of lifting that default assignment up as high as it can go is still a good one

so if we lift the default assignment of `''` to `ariaLabel` in the props destructure on line 143, the rest of the function has no worries about whether or not there's a valid value to deal with, and no imperative code has to be added to the HTML, and we can nicely just directly pass variables to the attrs of the `<a>` tag down here.

Reply:
@scottcreynolds Not nitpicky at all - I appreciate these kinds of comments. Helps me improve my thought process/knowledge. Btw totally agree, I went ahead and made the change you recommended.